[[Saccharomyces cerevisiae]]
[[Nile red]]
[[Lipid droplet]]
[[Adaptive evolution]]
[[Squalene]]
[[Next Generation Sequencing]]
[[Single-molecule Real-time Sequencing]]
[[Pectobacterium Spp.]]
[[Soft Rot Bacteria]]
[[Dickeya Spp.]]
[[Resistance gene]]
[[peritonitis]]
[[Bacteroides Fragilis]]
[[Intra-abdominal Infection]]
[[Drug resistance gene]]
[[Oxazolidinone]]
[[Multilocus Sequence Typing]]
[[Phenicol]]
[[Polyethylene]]
[[Alkb-like]]
[[Paenibacillus Aquistagni]]
[[Secondary metabolites]]
[[Erythrobacter]]
[[Pan-genome Analysis]]
[[Flanking sequence]]
[[Insertion Site]]
[[Event-specific Detection]]
[[Virus-resistant Transgenic Soybean]]
[[Surveillance]]
[[foodborne disease]]
[[Public Health Laboratory]]
[[Galaxytrakr]]
[[Ncbi-pd]]
[[Polyclonal Source]]
[[Oncogenes]]
[[Reprogramming]]
[[Pluripotent stem cells]]
[[Point mutations]]
[[enrichment]]
[[Endothelial Cells]]
[[Indels]]
[[Genomic Integrity]]
[[Parental Cells]]
[[Litter size]]
[[Genetic variation]]
[[Candidate gene]]
[[Goat (capra hircus)]]
[[Multidrug resistance]]
[[virulence factors]]
[[Stenotrophomonas]]
[[Antimicrobial Resistance Genes]]
[[Evolution]]
[[Mycobacterium tuberculosis]]
[[Genotyping]]
[[Clonal Complex]]
[[L4.3/lam]]
[[Pathogenicity]]
[[Vibrio parahaemolyticus]]
[[niche adaptation]]
[[Penaeus Vannamei]]
[[Recombination]]
[[Reassortment]]
[[Bayesian Phylodynamics]]
[[Tilapia Lake Virus]]
[[Streptococcus suis]]
[[Antibiotic Resistance]]
[[Snp-based Phylogenetics]]
